testzip.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
not sure if we should keep this file.

Reply:
Could you add a CI job that tests the zip install flow?

Reply:
Added a simple CI job that doesn't use this script. I'll remove this script unless there is interest in keeping it.

Reply:
I agree, this can go.